eip-2930.md

```diff
@@ -0,0 +1,132 @@
+---
+eip: 2930
+title: Optional access lists
+author: Vitalik Buterin (@vbuterin), Martin Swende (@holiman)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-08-29
+requires: EIP 2718; EIP 2929
+---
+
+## Simple Summary
+
+Adds a transaction type which contains an access list, a list of addresses and storage keys that the transaction plans to access. Accesses outside the list are possible, but become more expensive. Intended as a mitigation to contract breakage risks introduced by EIP 2929 and simultaneously a stepping stone toward broader use of access lists in other contexts.
+
+## Abstract
+
+We introduce a new [EIP-2718](https://github.com/ethereum/EIPs/pull/2718) transaction type, with the format `rlp([3, [nonce, gasPrice, gasLimit, to, value, data, access_list, senderV, senderR, senderS]])`.
+
+The `access_list` specifies a list of addresses and storage keys; these addresses and storage keys are added into the `accessed_addresses` and `accessed_storage_keys` global sets (introduced in https://notes.ethereum.org/@vbuterin/BkrNbeAfD). A gas cost is charged, though at a discount relative to the cost of accessing outside the list.
```

Comment:
Recommend against linking out to external documents whenever possible.  The links will likely *eventually* 404, at which point the EIP becomes harder to understand.  Where possible, either inline the content or add it as an asset in `../assets/eip-2930/...`